### ISSUE TITLE
Question: Button positioning in main folder list view

### ISSUE THREAD

**Issue by dataflake** [zopefoundation/Zope#537]
I'm dealing with many ZODB folders that have a large number of items in them. How hard would it be to make the buttons ("Rename", "Cut", "Copy", etc) always visible at the bottom of the page, no matter where you scroll to?

**Comment by drfho** [comment]
@dataflake 
hi, we could work with fixed position like this
![sticky_footer](https://user-images.githubusercontent.com/29705216/55687196-86885d80-596a-11e9-9812-13e7f0b92d3c.png)

CSS:
```
form[name="objectItems"] {
	margin-bottom:90px;
}
form[name="objectItems"] .form-group.zmi-controls {
    position: fixed;
    bottom: 0;
    height: 60px;
    line-height: 60px;
    background: rgba(252, 252, 252, 0.50)
}
```
Maybe we should add a CSS-Class for the extra-large lists (>50 items) and show the fixed control-set only in this case:
`<form name="objectItems" class="xl-list"...`
So the CSS will be like
```
form[name="objectItems"].xl-list { ... }
form[name="objectItems"].xl-list .form-group.zmi-controls { ... }
```
With that additional class name we could evaluate the UX case and maybe find a more elegant solution?
Please try it, does it help?
Cheers
fh

**Comment by dataflake** [comment]
@drfho thank you very much for the quick reply. I have implemented the second solution in #538 . This is exactly what I was thinking about.

The only thing I would change is how the part of the listing behaves that is covered by the button row. It should not shine through, the row should be fully opaque. Bonus points for a visible cue that the list continues behind those buttons, maybe a fade-out effect above the row of buttons.
